### PR TITLE
#11667: Skip failing SD and resnet tests on single-card WH regression

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket

#11667 #10922 #10923 

### Problem description

Skipping failing tests while owners (@mywoodstock and @esmalTT can take a look)

### What's changed

Skip the tests / increase timeouts

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

File: .github/workflows/fast-dispatch-full-regressions-and-models.yaml [hunk 1]
```diff
@@ -103,12 +103,19 @@ jobs:
               cmd: ./tests/scripts/run_tests.sh --tt-arch wormhole_b0 --pipeline-type frequent_api --dispatch-mode fast,
               timeout: 40
             },
+            {
+              name: "[Unstable] N150 models",
+              arch: wormhole_b0,
+              runs-on: ["cloud-virtual-machine", "arch-wormhole_b0", "N150", "in-service", "mount-cloud-weka"],
+              cmd: tests/scripts/single_card/nightly/run_wh_b0_unstable.sh,
+              timeout: 55
```
Comment (ttmchiou) [inline .github/workflows/fast-dispatch-full-regressions-and-models.yaml:111]: Are we adding this intentionally for patch missing test coverage of unstable wormhole?

Comment (tt-rkim) [reply .github/workflows/fast-dispatch-full-regressions-and-models.yaml:111]: Yes, @esmalTT requested in ticket #11370

**Comment by TT-billteng** [comment]
why do we need to skip if we're already using the pytest auto-reset infra?

**Comment by tt-rkim** [comment]
@TT-billteng yes and owners can work on it
nightly fd been red for time

**Review by TT-billteng — APPROVED** [review]

**Review by ttmchiou — APPROVED** [review]

**Review by esmalTT — APPROVED** [review]

**Review by mywoodstock — APPROVED** [review]

**Pull request merged**